deploy/SessionManager_0.js

```diff
@@ -0,0 +1,37 @@
+const {BigNumber} = require('@ethersproject/bignumber');
+
+const deploymentName = 'SessionsManager_0';
+const freeSessions = BigNumber.from('0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff'); // max uint256
+/**
+ * @param {import ('hardhat/types').HardhatRuntimeEnvironment}
+ */
+module.exports = async ({getNamedAccounts, deployments}) => {
+  const {deploy, get, execute} = deployments;
+  const {deployer, payoutWallet} = await getNamedAccounts();
+  const revvToken = await get('PolygonREVV');
+  await deploy(deploymentName, {
+    contract: 'SessionsManager',
+    from: deployer,
+    args: [
+      revvToken.address, // revvToken
+      payoutWallet, // payoutWallet
+    ],
+    log: true,
+  });
+  try {
+    await execute(
+      deploymentName,
+      {
+        from: deployer,
+        log: true,
+      },
+      'addFreeSessions',
+      freeSessions
+    );
+  } catch (e) {
+    // check if error is free sessions already added
+    if (!e.message.includes('Sessions: sessions overflow')) throw e;
+  }
```

Comment:
also need to set the session price to 1 REVV on this contract

done manually in https://amoy.polygonscan.com/tx/0x8469400abf5814fcb5d1dd98a5d97ec8da0a130de0ebafa9e4fdc7e3eab40ddf